backends/s3/s3.go

```diff
@@ -48,8 +46,8 @@ type Options struct {
 	// CreateBucket tells us to try to create the bucket
 	CreateBucket bool `yaml:"create_bucket"`
 
-	// EndpointURL can be set to something like "http://localhost:9000" when using Minio
-	// instead of AWS S3.
+	// EndpointURL can be set to something like "localhost:9000" when using Minio
```

Comment:
Please don't change the configuration format if not strictly necessary. This seemingly innocent change would break all existing configurations.

Reply:
Tested this against a configuration I had used prior and was presented with the following error (thrown by minIO client):
"Endpoint url cannot have fully qualified paths"

Endpoint used: https://s3.eu-central-1.amazonaws.com/

See suggestion in the 'Secure' flag comment, perhaps that can be used to prevent/solve both.

Reply:
This error was the reason for this change in the first place, but indeed the configuration format should not have been changed.